### PR TITLE
allow empty description in intervention lines 

### PR DESCRIPTION
**Pull request by bafbes**
Previously empty description in intervention lines leaded to input error "empty description". By defining to 1 the constant _FICHINTER_EMPTY_DESC_ABB_ description field would not appear on lines and empty descriptions would be stored.

### CODE REVIEW

**Comment by eldy** [comment]
Hard string constant like _FICHINTER_EMPTY_DESC_ABB_ are not allowed.
Can you replace with 
empty(global->$conf->FICHINTER_DO_REQUEST_DESC)

**Comment by bafbes** [comment]
Do FICHINTER_EMPTY_LINE_DESC do the trick ?

**Comment by bafbes** [comment]
fix faulting translation in products.lang

**Comment by bafbes** [comment]
Hi,
I finally fixed it correctly. 
The only problem is that I combined it with the pull request related to commit 5ee0fe4 : "fix faulting translation in products.lang"
I don't know how to separate this two pull requests and how to add a pull request while another pull request is in wait to be merged.

File: htdocs/fichinter/card.php [hunk 1]
```diff
@@ -961,7 +961,7 @@
 
 	$soc=new Societe($db);
 
-	print load_fiche_titre($langs->trans("AddIntervention"));
+	print_fiche_titre($langs->trans("AddIntervention"));
```
Comment (eldy) [inline htdocs/fichinter/card.php:964]: Why did you replace
print load_fiche_titre
with a deprecated function ?

File: htdocs/fichinter/card.php [hunk 2]
```diff
@@ -1525,7 +1525,7 @@
 					print '<td align="right">';
 					$selectmode='select';
 					if (! empty($conf->global->INTERVENTION_ADDLINE_FREEDUREATION)) $selectmode='text';
-					$form->select_duration('duration',$objp->duree,0, $selectmode);
+					$form->select_duration('duration',$objp->duree, $selectmode);
```
Comment (eldy) [inline htdocs/fichinter/card.php:1528]: This looks to be a regression.

Comment (bafbes) [inline htdocs/fichinter/card.php]: It's just an error in my repository.
It's corrected now.

2015-11-28 15:52 GMT+01:00 Laurent Destailleur <notifications@github.com>:

Comment (bafbes) [inline htdocs/fichinter/card.php]: I am really confused in what happened in my repository.
It seems that I merged accidentally some temporary branches.
I do not knoy how to resolve it.
Can you do it for me ?

2015-11-28 22:40 GMT+01:00 Abbes Bahfir <bafbes@gmail.com>:

**Comment by bafbes** [comment]
close for errors

**Pull request closed without merging**